save-backend/src/main/kotlin/org/cqfn/save/backend/configs/WebConfiguration.kt

```diff
@@ -31,4 +34,14 @@ class WebConfiguration {
             ok().contentType(TEXT_HTML).bodyValue(html)
         }
     }
+
+    /**
+     * @param builder
+     * @return web client
+     */
+    @Bean
+    fun webClient(builder: WebClient.Builder): WebClient =
```

Comment:
We should call it `preprocessorWebClient` because backend will query orchestrator too in the future